pkg/extraconfig/extraconfig.go

```diff
@@ -64,8 +67,16 @@ func GetExtraConfig() (*ExtraConfig, error) {
 
 	// build extraconfig object
 	extraConfig := ExtraConfig{}
-	if val, ok := extraConfigMap["loggingLevel"]; ok {
-		extraConfig.LoggingLevel = val
+	for key, val := range extraConfigMap {
+		switch key {
+		case "loggingLevel":
+			extraConfig.LoggingLevel = val
+		case "awsProfile":
+			os.Setenv("AWS_PROFILE", val)
```

Comment:
i feel we need to define the vars in extraconfig instead of set it as env var here. we can get it in the upper layer and set it as env var.

Reply:
I had the same thought but the style will be different with `loggingLevel` and we had the release for this change yesterday